app/code/Magento/Backend/Model/Widget/Grid/Parser.php

```diff
@@ -30,8 +30,9 @@ public function parseExpression($expression)
         $expression = trim($expression);
         foreach ($this->_operations as $operation) {
             $splittedExpr = preg_split('/\\' . $operation . '/', $expression, -1, PREG_SPLIT_DELIM_CAPTURE);
-            if (count($splittedExpr) > 1) {
```

Comment:
I assume that old check would not pass if `$splittedExpr` is and array with 1 element, while new one would pass.

Reply:
@ishakhsuvarov thank you for your review! You are totally right. Fix uploaded.